org.kde.krita.yaml

```diff
@@ -492,22 +521,29 @@ modules:
       - type: archive
         url: https://www.libraw.org/data/LibRaw-0.20.2.tar.gz
         sha256: dc1b486c2003435733043e4e05273477326e51c3ea554c6864a4eafaff1004a6
+        x-checker-data:
+          type: anitya
+          project-id: 1709
+          stable-only: true
+          url-template: https://www.libraw.org/data/LibRaw-$version.tar.gz
       - type: patch
         paths:
           - patches/libraw/0001-LibRaw-cmake.patch
           - patches/libraw/0002-Use-CMake-install-prefix.patch
   - name: gsl
-    buildsystem: cmake-ninja
```

Comment:
Looks weird. Are they no longer using cmake?

Reply:
The version used before at https://github.com/ampl/gsl/ includes the CMake build support but that isn't on the release monitor site looking it up so I switched it to the one they had at https://release-monitoring.org/project/1267/.